pkg/platform/baremetal/baremetal.go

```diff
@@ -143,58 +144,67 @@ func createTerraformConfigFile(cfg *config, terraformPath string) error {
 	t := template.New(tmplName)
 	t, err := t.Parse(terraformConfigTmpl)
 	if err != nil {
+		// TODO: Use template.Must().
 		return errors.Wrap(err, "failed to parse template")
 	}
 
 	path := filepath.Join(terraformPath, tmplName)
 	f, err := os.Create(path)
 	if err != nil {
-		return errors.Wrapf(err, "failed to create file %q", path)
+		return fmt.Errorf("creating file %q: %w", path, err)
 	}
+
 	defer f.Close()
 
 	// Configure oidc flags and set it to KubeAPIServerExtraFlags.
 	if cfg.OIDC != nil {
 		// Skipping the error checking here because its done in checkValidConfig().
 		oidcFlags, _ := cfg.OIDC.ToKubeAPIServerFlags(cfg.K8sDomainName)
-		//TODO: Use append instead of setting the oidcFlags to KubeAPIServerExtraFlags
+		// TODO: Use append instead of setting the oidcFlags to KubeAPIServerExtraFlags
```

Comment:
```suggestion
		// TODO: Use append instead of setting oidcFlags to KubeAPIServerExtraFlags.
```

I find "the" before a var name confusing.

Reply:
OK, though I didn't really touch that line, only because linter was complaining `//TODO:` vs `// TODO:` :smile:

Reply:
Not looking for suspects, just want to have cleaner errors while we're at it :-)